src/domain/app/apollo/apolloClient.ts

```diff
@@ -0,0 +1,62 @@
+import * as Sentry from '@sentry/browser';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloClient } from 'apollo-client';
+import { ApolloLink } from 'apollo-link';
+import { setContext } from 'apollo-link-context';
+import { onError } from 'apollo-link-error';
+import { HttpLink } from 'apollo-link-http';
+import { ErrorMessage } from 'formik';
+
+import { apiTokenSelector } from '../../auth/selectors';
+import i18n from '../i18n/i18nInit';
+import { store } from '../store';
+
+const cache = new InMemoryCache({
+  cacheRedirects: {
+    Query: {
+      placeDetails: (_, args, { getCacheKey }) =>
+        getCacheKey({ __typename: 'Place', id: args.id }),
+    },
+  },
+});
+
+const httpLink = new HttpLink({
+  uri: process.env.REACT_APP_API_URI,
+});
+
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors)
+    graphQLErrors.forEach(({ message, locations, path }) => {
+      const errorMessage = `[GraphQL error]: Message: ${message}, Location: ${locations}, Path: ${path}`;
+
+      Sentry.captureException(ErrorMessage);
+
+      if (process.env.NODE_ENV === 'development') {
+        // eslint-disable-next-line no-console
+        console.error(errorMessage);
+      }
+    });
+
+  if (networkError) {
+    Sentry.captureException(`[Network error]: ${networkError}`);
```

Comment:
Are you sure you want the network errors in Sentry? 

We hopefully have other ways to monitor availability of backend services, and we don't think we want to know if end users have network errors or not..